fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/serializers/RegisteredLookupDimensionToDimensionSpecSpec.groovy

```diff
@@ -0,0 +1,84 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.serializers
```

Comment:
There seems to be a typo in the file name, there are 2 `Spec`. Can we rename the file?